analytics/sensors/src/main/java/quarks/analytics/sensors/Filters.java

```diff
@@ -99,7 +99,7 @@ private Filters() {}
      * <LI>The first tuple.</LI>
      * </UL>
      * <BR>
-     * <img src="doc-files/deadband.png" alt="Deadband example"/>
+     * <img src="doc-files/deadband.png" alt="Deadband example">
```

Comment:
I think the original above is what's needed.  Was the image rendering wrong for you?

Thanks for many of the other fixes.

For what it's worth, it seems only the "<" needs to be escaped to avoid confusing browsers.  ">" is OK.  So for example, the changes to Connector below seem unnecessary.  Were you seeing a rendering problem?

Also &lt;pre> and/or &lt;code> provide some "protection" as well - e.g., line 157 in TStream below already renders correctly.

FWIW, I've taken to trying to use the following idiom as it seems "stuff" can then basically anything, particularly unescaped "<", so I can just write / copy code in its natural form
&lt;pre>{@code
stuff...
}&lt;/pre>